RadarSDK/RadarAPIClient.m

```diff
@@ -462,14 +462,24 @@ - (void)reverseGeocode:(CLLocation *)location completionHandler:(RadarGeocodeAPI
     }];
 }
 
-- (void)ipGeocode:(RadarIPGeocodeAPICompletionHandler)completionHandler {
+- (void)ipGeocode:(NSString *)ip completionHandler:(RadarIPGeocodeAPICompletionHandler)completionHandler {
     NSString *publishableKey = [RadarSettings publishableKey];
     if (!publishableKey) {
         return completionHandler(RadarStatusErrorPublishableKey, nil, nil);
     }
 
     NSString *host = [RadarSettings host];
-    NSString *url = [NSString stringWithFormat:@"%@/v1/geocode/ip", host];
+
```

Comment:
does client need to do some sanity check on the `ip` ?

Reply:
@kochis thoughts? I assume whatever validation we do on the FE won't be as comprehensive as https://www.npmjs.com/package/ip-address

I can try splitting on `.` or `:` and making sure there are > 3 elements, but I don't want to trigger any false negatives

Reply:
We do validation on the server so we should be ok to leave validation out on the client side:

https://github.com/radarlabs/server/blob/8bb83357266a95cdadef9a9040387b9a00e7d578/app.ts#L1363
https://github.com/radarlabs/server/blob/master/server/utils/Validate.ts#L240